Location fix

marcvaneijk opened the pull request:
Changed resource location from parameter to resourceGroup().location. Deleted location parameter from deployment template and parameter file.

singhkay commented:
@marcvaneijk error `        "error": "InvalidTemplateDeployment : The template deployment 'fakedDeploymentName' is not valid according to the validation procedure. The tracking id is '94f49ce1-e35f-444d-9e36-67c589ac627b'. See inner errors for details. Please see http://aka.ms/arm-deploy for usage details.\nInvalidDomainNameLabel : The domain name label uniqueDNS is invalid. It must conform to the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$.\n"`

marcvaneijk commented:
@singhkay 
- storageAccountName from param to var
- replaced disk endpoints to dynamic
- removed storageAccountName from param file
- updated param file with placeholders
